src/evd/evd.hpp

```diff
@@ -75,6 +85,8 @@ int evdOptions::initFromCmdLine(int argc, const char **argv)
     args::ValueFlag<int> yy(parser, "ysize", "Half window in y", {'y'});
     args::ValueFlag<int> mm(parser, "minNeighbors", "Minimum number of neighbors", {'n'});
     args::ValueFlag<int> kk(parser, "miniStackCount", "Mini-stack count", {'k'});
+    args::ValueFlag<std::string> tmethod(parser, "method", "Decomposition method - MLE / EVD/ SBAS", {'m'});
+    args::ValueFlag<int> bw(parser, "bandWidth", "SBAS bandwidth", {'b'});
```

Comment:
For historical reasons and to increase clarity, I suggest we use "StBAS" or "STBAS" which stands for Small-Temporal Baseline Subset, which is what we really support here and not SBAS which may refer to disconnected networks. StBAS is also consistent with recent publications by Ansari et al, which you have referred to.